.changeset/light-tables-drive.md

```diff
@@ -0,0 +1,38 @@
+---
+"@latticexyz/world-modules": patch
+---
+
+Fixed `ERC20Module` to register the `TotalSupply` table when creating a new token.
+
+You can register the table on existing Worlds with a Forge script:
```

Comment:
```suggestion
If you've deployed a world with the `ERC20Module`, we recommend patching your world to register this table so that indexers can properly decode its record. You can do so with a simple Forge script:
```